src/Gren/Kernel/Platform.js

```diff
@@ -48,14 +48,18 @@ function _Platform_initialize(
   var initPair = init(result.a);
   var model = initPair.__$model;
   var stepper = stepperBuilder(sendToApp, model);
-  var ports = _Platform_setupEffects(managers, sendToApp);
+  var ports = _Platform_setupEffects(managers, sendToApp, sendCmdToApp);
 
   function sendToApp(msg, viewMetadata) {
     var pair = A2(update, msg, model);
     stepper((model = pair.__$model), viewMetadata);
     _Platform_enqueueEffects(managers, pair.__$command, subscriptions(model));
   }
 
+  function sendCmdToApp(cmd) {
```

Comment:
This doesn't necessarily send anything to the application. Perhaps `executeCmd` works well for this function?